Tasks/FtpUpload/task.json

```diff
@@ -0,0 +1,97 @@
+{
+  "id": "6f8c69a5-b023-428e-a125-fccf4efcb929",
+  "name": "FtpUpload",
+  "friendlyName": "Ftp Upload",
+  "description": "Ftp Upload",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "Upload files to an FTP server using either FTP, or secure FTPS",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [],
+  "version": {
+    "Major": "1",
+    "Minor": "0",
+    "Patch": "0"
+  },
+  "instanceNameFormat": "FTP Upload: $(rootFolder)",
+  "groups": [
+        {
+          "name": "advanced",
+          "displayName": "Advanced",
+          "isExpanded": false
+        }
+    ],
+  "inputs": [
+    {
+      "name": "serverEndpoint",
+      "type": "connectedService:Generic",
+      "label": "FTP Service Endpoint",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "Select the service endpoint for your FTP server.  To create one, click the Manage link and create a new Generic Service Endpoint, enter the FTP server URL for the server URL, e.g. ftps://server.example.com, and required credentials."
```

Comment:
Even though it's pretty obvious, can this help Markdown mention that ftp:// addresses will use FTP, but ftps:// URLs will use secure FTPS?